tasks/community-modules.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Copy requirements.txt
   copy: # 'noqa' below disables [E404] Don’t compare to empty string
-    src: "{{ inventory_dir }}/../files/requirements.txt" # noqa 404
+    src: "{{ odoo_role_community_modules_requirements_path }}" # noqa 404
```

Comment:
Is the `# noqa 404` comment still needed with this change? Maybe @danypr92 knows.

Reply:
> Is the `# noqa 404` comment still needed with this change? Maybe @danypr92 knows.

Nop, we can remove the comment and the comment of the previous line.

Reply:
@warpet would you mind to fix it?

Reply:
sure. did the modification